modules/json/src/main/java/org/mule/module/json/transformers/TransformerInputs.java

```diff
@@ -120,6 +124,8 @@ public static SchemaFactory createSchemaFactory(String schemaLanguage)
             schemaFactory = SchemaFactory.newInstance(schemaLanguage);
         }
 
+        XMLSecureFactories.createDefault().configureSchemaFactory(schemaFactory);
```

Comment:
Same as createTransformerFactory. It also includes an additional parameter for constructing.